index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <h2>Welcome</h2>
+    <nav>
+      <span id="humburger"><i class="fa humburger fa-bars"></i></span>
+    </nav>
+    <ul class="nav-bar-list">
+      <li><a href="home" aria-label="Nav item"></a></li>
+      <li><a href="about" aria-label="Nav item"></a></li>
+      <li><a href="contact" aria-label="Nav item"></a></li>
+    </ul>
```

Comment:
- [ OPTIONAL] Great job while implementing this. I appreciate a lot the way you structured your layout and by following this requirement. Just look this out, so what do you if the nav-bar-list to be inside the ` nav` tag? as long as it is still in the navigation menu.
   ```HTML
   <nav>
     <ul>
      <li>item1</li>
      <li>item2</li>
      <li>item3</li>
    </ul>
   <button><i class="fa humburger fa-bars"></i></button>
  </nav>
   ```